Library/Homebrew/cmd/update-bash.sh

```diff
@@ -289,10 +301,16 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  DIRS=("$HOMEBREW_LIBRARY"/Taps/*/*)
+  if [[ -z "$SKIP_UPDATE_HOMEBREW_REPOSITOR" ]]
+  then
+    DIRS=("$HOMEBREW_REPOSITORY" "${DIRS[@]}")
+  fi
```

Comment:
I'm all for simple code as long as there is no compromise  in function and robustness .